cl/recap/migrations/0003_emailprocessingqueue_message_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.7 on 2021-08-28 11:15
```

Comment:
Can you merge this migration with 0004, please? If you migrate backwards to 0002, then do `makemigrations` again, it should give you a single file. You could also just hack these two files together by hand, but if you do, make sure that your migration history in your database doesn't get confused about where 0004 went.

Reply:
Just tweaked this comment to include better instructions, italicized. (You have to delete the files before it'll make new ones.)